bootstrap/kubeadm/config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-        - image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:main
+        - image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller-arm64:dev
```

Comment:
We should keep this the same, same for manager_pull_policy
\+ in other providers / including Runtime Extension provider

Reply:
We should probably document why we have those specific values:
1. image: that we can run e2e tests locally without having to build Docker images
2. pull policy: that always the latest image is pulled & used

Reply:
I have rolled back those changes (leftovers from running tests locally). We keep consistency with other providers